### PR TITLE
fix: optimize helm script

### PR DESCRIPTION
**Pull request by lintingbin**
## Brief change log
1. Remove command: ["bash","-c","bash ./bin/streampark.sh start_docker"] to fix the issue where it cannot start.
2. Add volume-related configurations to facilitate users to insert their own k8s storage configurations.
3. Add TLS-related configurations to ingress.
4. Optimize RBAC configurations by adding edit permissions for the default serviceaccount in the namespace, to prevent permission issues for new users when starting Flink.

## Verifying this change

This change is a trivial rework / code cleanup without any test coverage.

## Does this pull request potentially affect one of the following parts
 - Dependencies (does it add or upgrade a dependency): no

### CODE REVIEW

File: helm/streampark/templates/streampark.yml [hunk 1]
```diff
@@ -60,7 +60,6 @@ spec:
             {{- toYaml .Values.spec.container.env | nindent 12 }}
           securityContext:
             privileged: false
-          command: ["bash","-c","bash ./bin/streampark.sh start_docker"]
```
Comment (wolfboys) [inline helm/streampark/templates/streampark.yml:63]: Can you please explain why remove start_docker?

Comment (lintingbin) [reply helm/streampark/templates/streampark.yml:63]: https://github.com/apache/incubator-streampark/blob/dev-2.1.5/docker/Dockerfile#L79

Comment (lintingbin) [reply helm/streampark/templates/streampark.yml:63]: Remove command: ["bash","-c","bash ./bin/streampark.sh start_docker"] to fix the issue where it cannot start.

Comment (wolfboys) [reply helm/streampark/templates/streampark.yml:63]: Thanks for your clarification

**Review by wolfboys — APPROVED** [review]
LGTM

**Pull request merged**